File: Java/1158.java
import java.util.Scanner;

public class a1158 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int N, X, Y, soma = 0;
        N = input.nextInt();

        for (int i = 1; i <= N; i++) {
            

            X = input.nextInt();
            Y = input.nextInt();

            if (X % 2 == 0) {
                X++;
            }

            for (int K = 0; K <= Y-1; K++) {
                soma+=X;
                X+=2;
                
            }
            
            System.out.println(soma);
            soma = 0;
        }
    }
}

File: 1158.py
N = int(input())

for i in range (1,N+1):
    soma = 0
    X , Y = list(map(int,input().split()))

    if X % 2 == 0:
        X+=1

    for k in range (Y):
       soma +=X
       X +=2
  
       
    print(soma)

    
   

